src/components/FilterByCentury/CenturyButton.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import { SearchLink } from '../SearchLink/SearchLink';
+import React from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+type Props = {
+  century: string;
+};
+
+export const CenturyButton: React.FC<Props> = ({ century }) => {
+  const [searchParams] = useSearchParams();
+  let paramsCenturies = [...searchParams.getAll('centuries')];
```

Comment:
There are more elegant ways to solve this problem than using let to store an array. For example, using a helper function that will return a new array and write it to the reference parameters. Let for an array is almost always not the best idea.